package exception;

/**
 * @author janemin
 * @create 18-9-1
 */
public class ExcepiontDemo {
    public static void main(String[] args) {

/*        int i = 12;
        int j = 0;
        try {
            System.out.println((i / j));

        } catch (ArithmeticException e) {
            e.printStackTrace();
            System.out.println(e);
        }
        System.out.println("program end");*/
    }

    public static int test() {
        Scanner
    }
}
